includes/class-boldgrid-editor-assets.php

```diff
@@ -163,6 +163,31 @@ public static function is_webpack() {
 		return ( defined( 'BGEDITOR_SCRIPT_DEBUG' ) && BGEDITOR_SCRIPT_DEBUG );
 	}
 
+	/**
+	 * Maybe Enqueue Animate.css
+	 *
+	 * Checks if post uses the animate.css or not.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return bool True if animate.css is to be enqueued.
+	 */
+	public function maybe_enqueue_animate() {
```

Comment:
Should we make sure it's a "page" or "post"? e.g. we don't need to have it loaded on content the P&PB isn't used to manage

Reply:
P&PB is able to manage custom post types as well, not just pages and posts.